src/azul/lambdas.py

```diff
@@ -132,13 +134,15 @@ def manage_lambda(self, lambda_name: str, enable: bool):
 
                 if original_concurrency_limit is not None:
                     log.info(f'Setting concurrency limit for {lambda_name} back to {original_concurrency_limit}.')
+                    # Concurrency settings apply to the function as a whole,
+                    # including all published versions and the unpublished
+                    # version
```

Comment:
```suggestion
                    # Concurrency settings apply to the function as a whole,
                    # including all published versions and unpublished
```

Or try to minimize the lines, make the comment as concise possible.

Reply:
I originally thought the comments would be useful to explain why we specify the Lambda alias in some boto3 calls (e.g. `list_event_source_mappings`) and not others (e.g. `put_function_concurrency`), but I agree it's a bit redundant since it's available from boto3's documentation. I will remove.